src/redux/reducers/componentConfig/initialState.js

```diff
@@ -754,7 +754,18 @@ const initialPlotConfigStates = {
 const initialComponentConfigStates = {
   interactiveHeatmap: interactiveHeatmapInitialConfig,
 };
-export { initialPlotConfigStates, initialComponentConfigStates };
+
+const initialPlotDataState = {
```

Comment:
where was the initial plot data defined before? is this supposed to be reused (is it already?) by other plots? does it match the other plots' initial data state?

Reply:
`plotData` was initially added as the payload in `loadPlotConfig` when data is being fetched from the server. `plotData` is used to pass data from the server to the plots. Not all plots have `plotData`. Right now, only plots in the filters in `data-processing` uses `plotData`.

Reply:
I have modified `loadPlotConfig` to make it. I have also inserted `initialPlotDataState` when loading plot config. This way, all plots have the same states, although they might not use it.